web/src/OverviewTable.tsx

```diff
@@ -168,14 +180,18 @@ const ResourceTableHead = styled.thead`
 
 export const ResourceTableRow = styled.tr`
   border-top: 1px solid ${Color.gray40};
+  border-left: 4px solid transparent;
```

Comment:
this does add a small gap to the table rows' bg and top border. 🥲 one (sort of) fix i can think of is moving this border to `tbody > tr` and making the border color the same color as the row when it's not selected.

![Screen Shot 2022-03-11 at 2 12 11 PM](https://user-images.githubusercontent.com/23283986/157935870-2cdb5d68-c08d-432b-8b1f-3d4261aa8b68.png)

Reply:
fixed